### ISSUE TITLE
Overridden method unused param

### ISSUE THREAD

**Issue by maelstraggiotti** [Riverside-Software/sonar-openedge#440]
Hi,
Should parameters of an overridden method be excluded from the Unused parameter rule ? Could it be an option ?
Given this example : 
```
CLASS MyBase: 
    METHOD PUBLIC VOID DoSomething(
        INPUT piParam AS CHARACTER):
    END METHOD.
END CLASS.
```
```
CLASS MyInheritor INHERITS MyBase:
    METHOD PUBLIC OVERRIDE VOID DoSomething(
        INPUT piParam AS CHARACTER):
        /* Do something */
    END METHOD.
END CLASS.
```
piParam is indeed unused but we can't get rid of it. 

Thanks

**Comment by gquerret** [comment]
I think the cases described in the [Java rule](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS1172) may also apply in the OpenEdge rule

**Comment by maelstraggiotti** [comment]
Hi Gilles, 
Any ETA for this issue resolution ? Also, I think it will apply to implemented interfaces too ?
Thanks

**Comment by gquerret** [comment]
Late answer, sorry 😰
That will most likely be included in next release (end of February). Otherwise end of March.

**Comment by gquerret** [comment]
Fixed in upcoming 2.1.5.
An issue has however been found in the way methods are populated in the parse unit, which will result in false negatives. Fix will be for 2.1.6.
